change/@fluentui-react-charting-b1c51226-7ca4-40eb-9521-2b53b639045e.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29876&commitId=a374f5ea31de038f1a38580def6f36c806bd2d3e&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-VerticalBarChart</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Dynamic - Wrap Labels - RTL.default.chromium.png|4318| Changed|
|react-charting-VerticalBarChart.Rotated Label- Hide Legends - RTL.default.chromium.png|3324| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
The PR fixes the issue reported in the baseline screenshot.